config/containers/logging/local.md

```diff
@@ -0,0 +1,70 @@
+---
+description: Describes how to use the local logging driver.
+keywords: local, docker, logging, driver
+redirect_from:
+- /engine/reference/logging/local/
+- /engine/admin/logging/local/
+title: Local File logging driver
+---
+
+
+The `local` logging driver captures output from container's stdout/stderr and
+writes them to a local file on disk. The format of these files is unspecified.
```

Comment:
WDYM by "format is unspecified" ?

Reply:
Trying to convey that we are not going to document the format for public consumption because we don't want people messing with these files.
Also that the format itself may change over time.

Reply:
Not sure if that detail is necessary here. If you feel strongly about it, I'm fine.

Reply:
I think so because currently people parse the json logs and it makes it impossible to change the internal structure without breaking people.

Reply:
Thinking a bit more. If we are expecting to change the format and dont provide any tools for parsing, do you think we should make it `experimental`?

Reply:
I don't. It's the users choice to use it or not, gating it behind experimental just means no one will use it.

Reply:
Let's get @thaJeztah thought's on this